src/collectors/fio/fio.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
+
+"""
+The FioCollector collects results from fio runs
+
+#### Dependencies
+
+ * fio
+
+"""
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+import re
+import subprocess
+
+import diamond.collector
+
+
+def flatten_dictionary(input, sep='.', prefix=None):
```

Comment:
Looks like you copied this code and changed it.  Any reason you no longer use prefix in the code? See ceph collector code.